### ISSUE TITLE
Flow > 0.102.0 doesn't error on undefined types anymore, but only in flow-typed

### ISSUE THREAD

**Issue by julienw** [flow-typed/flow-typed#3825]
I was working on a new libdef and got the weirdest problem.
I reduced to that:
```js
declare module 'idb' {
  declare export function openDB(): Promise<DB>;
}
```
It's in a file `idb_v5.x.x/flow_v0.84.x-/idb_v5.x.x.js` without any test file.

Now when I run the tests with latest flow version, this doesn't error. This should error on `DB` because that type doesn't exist.
But when I run the tests with older flow versions, this was erroring until flow 0.102.0.
More weirdly, [in Flow Try](https://flow.org/try/#0CYUwxgNghgTiAEBbA9sArhBByAlsARlvAN4BQ88oksCIAHgA7IwAu8AZmgHZgs7Jd4yBiC4ARAEIAKAJQAueAAUYyRDgDOIADySAfAG5SAX1JA) this errors properly.

This is how I run tests.
Only latest:
```
(cd cli && node dist/cli.js run-tests --numberOfFlowVersions 1 idb_v5)
```
A lot of Flow versions:
```
(cd cli && node dist/cli.js run-tests --numberOfFlowVersions 100 idb_v5)
```

I'm glad older flows gave me the error I had. But I find it very strange that newer flows don't error. Any idea?

**Comment by STRML** [comment]
I am seeing the same issue. The currently committed "@sentry/browser" libdef contains a reference to a type that does not exist:

https://github.com/flow-typed/flow-typed/blob/f191a63cd206609ee8a6e0a81f9153dc8082c84e/definitions/npm/%40sentry/browser_v5.x.x/flow_v0.84.x-/browser_v5.x.x.js#L41

This only became evident after including it into a real project. Tests will pass, no matter what you do. You can put complete garbage in as that parameter type (seriously, just press on your keyboard) and the tests will pass. But if it resolves to a real type (e.g. `number`) and you pass a bool, a type error will correctly be shown.

**Comment by STRML** [comment]
In some subsequent testing I can also confirm that Flow <= 0.102.0 errors correctly:

```
    Error ------------------------------------------------------------------------------------------ browser_v5.x.x.js:41:33

    Cannot resolve name `SpanContext`.

       41|         startSpan(span?: Span | SpanContext, forceNoChild?: boolean): Span;
                                           ^^^^^^^^^^^
```

but newer versions do not.

**Comment by AndrewSouthpaw** [comment]
I haven't done a lot of digging into how the libdef tests are run, but my understanding is that at its core it's just running `flow` again. I wonder if it's something worth raising with the Flow team as well? The fact that when you try to use the libdef it does correctly raise an error is also pretty strange...

**Comment by julienw** [comment]
I had another quick look, and indeed it seems I reproduce the error in a project too. I didn't see it before because my main project is still using an older flow version (upgrades can be painful in big projects :) ).
Everything happens as if the undefined value is an any. This is concerning :/

What is different in try?

**Comment by STRML** [comment]
It's very odd. I've taken apart the test scripts and I can't figure out why it's doing it either. If I had the time, I would try modifying it to not import default libdefs, find a minimal reproduction, then run Flow in debug mode. Perhaps something it is including to prep the suite is triggering a new bug.

**Comment by Brianzchen** [comment]
This error appears to have fixed itself since 0.143.0. I've tested against the last 100 versions and you can see from the logs there's no errors before 0.143.0 but recent versions look good.

```
Running definition tests against latest 100 flow versions in /Users/brianchen/projects/flow-typed/definitions/npm...

Fetching all Flow binaries...
Finished fetching Flow binaries.

Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.104.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.105.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.105.2)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.106.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.106.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.106.2)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.106.3)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.107.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.108.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.109.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.110.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.110.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.111.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.111.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.111.2)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.111.3)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.112.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.113.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.114.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.115.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.116.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.116.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.117.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.117.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.118.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.119.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.119.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.120.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.120.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.121.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.122.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.123.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.124.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.125.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.125.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.126.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.126.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.127.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.128.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.129.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.130.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.131.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.132.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.133.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.134.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.135.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.136.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.137.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.138.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.139.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.140.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.141.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.142.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.143.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.143.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.144.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.145.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.146.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.147.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.148.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.149.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.150.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.150.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.151.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.152.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.153.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.154.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.155.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.155.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.156.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.157.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.158.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.159.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.160.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.160.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.160.2)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.161.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.162.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.162.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.163.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.164.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.165.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.165.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.166.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.166.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.167.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.167.1)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.168.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.169.0)...
Testing keymirror_v0.1.x/flow_v0.104.x- (flow-v0.170.0)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.98.0)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.98.1)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.99.0)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.99.1)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.100.0)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.101.0)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.101.1)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.102.0)...
Testing keymirror_v0.1.x/flow_v0.33.x-v0.103.x (flow-v0.103.0)...

ERROR: keymirror_v0.1.x/flow_v0.104.x-
* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.143.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.143.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.144.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.145.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.146.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.147.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.149.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.150.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.148.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.150.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.154.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.155.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.151.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.153.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.152.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.159.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.158.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.156.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.155.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.157.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.161.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.162.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.160.2): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.160.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.160.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.163.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.162.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.164.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.165.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.165.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.167.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.166.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.167.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.166.1): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.168.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.169.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error


* keymirror_v0.1.x/flow_v0.104.x- (flow-v0.170.0): Unexpected Flow errors (2):

    Error ----------------------------------------------------------------------------------------- keymirror_v0.1.x.js:2:38

    Cannot resolve name `DB`. [cannot-resolve-name]

       2|   declare function openDB(): Promise<DB>;
                                               ^^



    Found 1 error
```
